packages/core/src/jsx-runtime.js

```diff
@@ -0,0 +1,174 @@
+/* eslint-disable no-unused-vars */
```

Comment:
This file is copied almost exactly from `jsx.js` but I did what I *think* was necessary to replace calls to `React.createElement` with `jsx` from `react/jsx-runtime`.

Since most of the code doesn't care about the factory method, it should definitely factored to support both.